Upstream assumes using new wine64 implementation, which we only enable for musl atm.

Install dlls to both locations for now.

From 0930d9cfddbe43a3e3ed4ca29acdb87748570d9e Mon Sep 17 00:00:00 2001
From: "S. Christian Collins" <s_chriscollins@hotmail.com>
Date: Fri, 17 Nov 2023 00:27:39 -0600
Subject: [PATCH] fix 32-bit DLL location for multiarch Wine

---
 wineasio-register | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/wineasio-register b/wineasio-register
index 862fb0e..c7fbe73 100755
--- a/wineasio-register
+++ b/wineasio-register
@@ -34,7 +34,11 @@ u64=(
 for u in ${u32[@]}; do
     w=$(echo ${u} | sed -e 's|/i386-unix/wineasio32.dll.so|/i386-windows/wineasio32.dll|g')
     if [ -e "${u}" ] && [ -e "${w}" ]; then
-        cp -v "${w}" "${WINEPREFIX}/drive_c/windows/system32"
+        if command -v wine64 > /dev/null && [ -d "${WINEPREFIX}/drive_c/windows/syswow64" ]; then
+            cp -v "${w}" "${WINEPREFIX}/drive_c/windows/syswow64"
+        else
+            cp -v "${w}" "${WINEPREFIX}/drive_c/windows/system32"
+        fi
         regsvr32 "${u}"
         break
     fi
